pulsar-broker-common/src/main/java/org/apache/pulsar/common/configuration/PulsarConfigurationLoader.java

```diff
@@ -187,7 +187,8 @@ public static ServiceConfiguration convertFrom(PulsarConfiguration conf, boolean
                 try {
                     confField.setAccessible(true);
                     Field convertedConfField = ServiceConfiguration.class.getDeclaredField(confField.getName());
-                    if (!Modifier.isStatic(convertedConfField.getModifiers())) {
+                    if (!Modifier.isStatic(convertedConfField.getModifiers())
+                            && convertedConfField.getDeclaredAnnotation(FieldContext.class) != null) {
```

Comment:
Does this mean that the `properties` in the `conf` won't be appeared in the returned ServiceConfiguration?

Reply:
fixed.